test/dom/index.js

```diff
@@ -28,24 +82,167 @@ describe('dom', () => {
       },
     };
 
-    assert.deepEqual(utils.offset(), { top: 0, left: 0 }, 'empty call');
-    assert.deepEqual(utils.offset(node1), { top: 10, left: 10 }, 'without parent');
-    assert.deepEqual(utils.offset(node2), { top: 20, left: 20 }, 'with parent');
+    assert.deepEqual(offset(), { top: 0, left: 0 }, 'empty call');
+    assert.deepEqual(offset(node1), { top: 10, left: 10 }, 'without parent');
+    assert.deepEqual(offset(node2), { top: 20, left: 20 }, 'with parent');
+  });
+
+  it('documentOffset', () => {
+    const node1 = {
+      top: 10,
+      left: 100,
+      getBoundingClientRect() {
+        return { top: this.top, left: this.left };
+      },
+    };
+    const docContainer1 = {
+      pageXOffset: 15,
+      pageYOffset: 11,
+    };
+
+    const node2 = {
+      top: 40,
+      left: 20,
+      getBoundingClientRect() {
+        return { top: this.top, left: this.left };
+      },
+    };
+    const docContainer2 = {
+      pageXOffset: 15.8,
+      pageYOffset: 11.2,
+    };
+
+    assert.deepEqual(documentOffset(docContainer1, node1), { left: 115, top: 21 }, 'returns correct values');
```

Comment:
don't test the exact value as it might be flaky due to rounding, just test that it is not the same as the original